src/Engine/Manager/RenderManager.cpp

```diff
@@ -138,6 +147,8 @@ void RenderManager::Render(World& world, bool soundSources, bool particleEmitter
                     const glm::mat4 eyeTranslation = Managers().vrManager->GetHMDHeadToEyeMatrix(nEye);
                     const glm::mat4 eyeViewMatrix = eyeTranslation * lensViewMatrix;
                     const glm::mat4 projectionMatrix = headset->GetHMDProjectionMatrix(nEye, lens->zNear, lens->zFar);
+                    const glm::vec3 position = camera->GetWorldPosition();
+                    const glm::vec3 up(lensViewMatrix[0][1], lensViewMatrix[1][1], lensViewMatrix[2][1]);
```

Comment:
Same here with up vector

Reply:
const glm::vec3 up(viewMatrix[0][1], viewMatrix[1][1], viewMatrix[2][1]); is faster